### PR TITLE
Update xml-config-file.md

### PR DESCRIPTION
**Pull request by sammiili**
Updating Noise Suppression XML setting per the recent changes to switch from dropdown options to a toggle switch (on/off) for the Noise Suppression setting.

### CODE REVIEW

File: Teams/rooms/xml-config-file.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ If a variable value is of the wrong type, elements are out of order, elements ar
...
 | `<EnablePublicPreview>` | Boolean &#x2777; | First &#x2776; | Disabled by default. If true, public preview is enabled and end-users can access features in public preview on enabled Teams Rooms. See [Public preview for Microsoft Teams Rooms on Windows](../public-preview-doc-updates.md#public-preview-for-microsoft-teams-rooms-on-windows) for more information. |
-| `<NoiseSuppressionDefault>` | String | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> Auto (default). Teams decides on the best level of noise suppression based on local noise.</li><li><b>2</b> Low. Suppresses low levels of persistent background noise, such as a computer fan or air conditioner.</li><li><b>3</b> High. Suppresses all background sound that isn't speech.</li></ul> |
+| `<NoiseSuppressionDefault>` | String | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> High. Suppresses all background noises (stationary and non-stationary) that isn't speech.</li></ul> |
```
Comment (JohanFreelancer9) [inline Teams/rooms/xml-config-file.md:170]:
```suggestion
| `<NoiseSuppressionDefault>` | String | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> High. Suppresses all background noises (stationary and non-stationary) that aren't speech.</li></ul> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sammiili, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by sammiili** [comment]
hi @dstrome, following up if there is anything pending on my side for this to be published?

**Comment by scanum** [comment]
@dstrome
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**